tideways/provision.sh

```diff
@@ -123,4 +127,8 @@ install_xhgui_frontend
 enable_tideways_by_site
 restart_php
 
-echo " * Tideways and XHGui installed"
+if [[ ! -f "/srv/www/default/xhgui/composer.lock" ]]
+    echo " * XHGUI installtion failed!"
```

Comment:
XHGUI has a composer lock file in the repo, this check will never work https://github.com/perftools/xhgui/blob/master/composer.lock

Reply:
```suggestion
    echo " * XHGUI installation failed!"
```